### DISCUSSION TITLE
Dockerfile for building images?

### DISCUSSION THREAD

**Discussion by cpretzer** [firefly-iii/firefly-iii#6894] [General]
Hi,

I'm interested in building container images from source and wondering where I can find Dockerfiles to do so. Are those available?

**Comment by JC5** [comment]
Follow the links: https://github.com/firefly-iii/docker

**Reply by cpretzer** [reply]
Thanks! I was looking in the `firefly-iii/firefly-iii` repo and didn't think to look in a different org repo.
